plone/volto
# [WIP] Prototype for image crop

How to test this feature
1. create a news item.
2. select the image field.
you are able to see the crop image feature just select the area which you needed and click on save.
